backend/createStaffAndRooms.py

```diff
@@ -45,81 +46,101 @@ def search_ticket(ticket, guest_entries):
     return False
 
 
-def real_date(a_date):
+def real_date(a_date: str, year=None):
```

Comment:
loving the pydoc, ty 💖 

Is it possible to just mix this parsing into `pydantic` at all? I feel like over time we'll want some low-key NLP date parsing, and it would be nice to leverage the new validation/parsing framework for that.

Reply:
Yup, i think so. I want to get the validator in so it's not blocking other code, and then we can do any bool or date processing in the validator.